doc/source/troubleshoot/troubleshoot.rst

```diff
@@ -0,0 +1,441 @@
+    PS [System.Environment]::SetEnvironmentVariable('CADOE_LIBDIR222','C:\Program Files\ANSYS Inc\v222\CommonFiles\Language\en-us',[System.EnvironmentVariableTarget]::User)
+    PS echo $env:CADOE_LIBDIR222
+    C:\Program Files\ANSYS Inc\v222\CommonFiles\Language\en-us
+
+
+
+*****************
+Launching PyMAPDL
+*****************
+
+Even if you are able to correctly launch MAPDL, PyMAPDL might have some problems to launch
+MAPDL.
+
+
+
+Manually Set the Executable Location
+====================================
+If you have a non-standard install, ``pymapdl`` may be unable find
+your installation.  If that's the case, provide the location of MAPDL
+as the first parameter to ``launch_mapdl``.  For example, on Windows,
+this will be:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> exec_loc = 'C:/Program Files/ANSYS Inc/v211/ansys/bin/winx64/ANSYS211.exe'
+    >>> mapdl = launch_mapdl(exec_loc)
+
+For Linux:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    >>> exec_loc = '/usr/ansys_inc/v211/ansys/bin/ansys211'
+    >>> mapdl = launch_mapdl(exec_loc)
+
+Should this fail to launch or hang while launching, pass
+``verbose_mapdl=True`` when using ``launch_mapdl``.  This will print
+the output of MAPDL within Python and can be used to debug why MAPDL
+isn't launching. On Windows, output is limited due to the way
+MAPDL launches.
+
+
+Default Executable Location
+===========================
+
+The first time that you run PyMAPDL, it detects the
+available ANSYS installations.
+
+On Windows, Ansys installations are normally under:
+
+.. code:: text
+
+    C:/Program Files/ANSYS Inc/vXXX
+
+On Linux, Ansys installations are normally under:
+
+.. code:: text
+
+    /usr/ansys_inc/vXXX
+
+If PyMAPDL finds a valid ANSYS installation, it caches its
+path in the configuration file, ``config.txt``, whose path is shown in the
+following code:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core.launcher import CONFIG_FILE
+    >>> print(CONFIG_FILE)
+    'C:\\Users\\user\\AppData\\Local\\ansys_mapdl_core\\ansys_mapdl_core\\config.txt'
+
+
+In certain cases, this configuration might become obsolete. For example, when a new
+Ansys version is installed and an earlier installation is removed.
+To update this configuration file with the latest path, use:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core import save_ansys_path
+    >>> save_ansys_path(r"C:\Program Files\ANSYS Inc\v222\ansys\bin\winx64\ansys222.exe")
+    'C:\\Program Files\\ANSYS Inc\\v222\\ansys\\bin\\winx64\\ansys222.exe'
+
+If you want to check which Ansys installations PyMAPDL has detected, use:
+
+.. code:: python
+
+    >>> from ansys.mapdl.core.launcher import _get_available_base_ansys
+    >>> _get_available_base_ansys()
+    {222: 'C:\\Program Files\\ANSYS Inc\\v222',
+    212: 'C:\\Program Files\\ANSYS Inc\\v212',
+    -222: 'C:\\Program Files\\ANSYS Inc\\ANSYS Student\\v222'}
```

Comment:
```suggestion
    {222: 'C:\\Program Files\\ANSYS Inc\\v222',
     212: 'C:\\Program Files\\ANSYS Inc\\v212',
     222: 'C:\\Program Files\\ANSYS Inc\\ANSYS Student\\v222'}
```

Reply:
This regarding this.

Reply:
Ah, cool. Could you make a note of that?

Reply:
Yes, at the function and at the docs.